source/includes/quick-start/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	uri = os.Getenv("MONGODB_URI")
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
+	defer cancel()
+
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(uri))
+	if err != nil {
+		log.Panic(err)
+	}
+
+	defer func() {
+		if err = client.Disconnect(ctx); err != nil {
+			log.Panic(err)
+		}
+	}()
+
+	coll := client.Database("sample_mflix").Collection("movies")
```

Comment:
@terakilobyte it might be worth adding the note blurb from the findOne page (https://github.com/mongodb/docs-golang/pull/2/files#diff-5e814de2f029ecad71b3ce40d12e14dcc1319d1f8d55d45efb21d2212ed230b6R1) to call out to users that they need to load the sample data

Reply:
I did add wording and a link to the load sample dataset step in the setup instructions, but adding another note may help users who weren't paying attention.

Reply:
Thanks! I had not realized I suggested a self link :joy: